geometry_msgs/PoseStamped[] initial_poses
nav_msgs/OccupancyGrid map
float32 tool_width
int8 num_robots
---
bool error
nav_msgs/OccupancyGrid coverage_map
nav_msgs/Path[] coverage_paths